ci/env/install-bazel.sh

```diff
@@ -121,3 +121,7 @@ if [[ "${CI-}" == "true" ]]; then
     fi
   fi
 fi
+
+# Append bazel executable directory to system `PATH`.
+BAZEL_EXEC_BIN=$(dirname "$(which bazel)")
+echo "export PATH=${BAZEL_EXEC_BIN}:\$PATH" >> ~/.bashrc
```

Comment:
I'm not sure if it's good idea to change user's `.bashrc`. Should we instead update the doc instead telling people to put the installation path to the `PATH`?

Reply:
Sure that's good!

Reply:
TBH I still prefer to add it into system executable path, and it's commonly used in a lot of tools that I use:
- fzf: https://github.com/junegunn/fzf
- thefuck: https://github.com/nvbn/thefuck

```
[ -f ~/.fzf.bash ] && source ~/.fzf.bash
eval "$(thefuck --alias)"
```

Reply:
I feel it might have unintended side effect: at least we should do `PATH=:$PATH:${BAZEL_EXEC_BIN}` instead of `PATH=${BAZEL_EXEC_BIN}:\$PATH`

Reply:
Emm I don't see much difference? Just search order. BTW in the latest commit there's only Readme doc change but no code change